### PR TITLE
refactoring variable names + minor fixes

### LINKED ISSUE #91: Front end variable standarisation
We have a crazy mixture of variable name styles (my fault). Let's choose one style and refactor.

### PR DESCRIPTION
**Pull request by crangelsmith**
Branched out of PR #90 , we can merge this into that one before merging PR #90 into develop.

Fixes #91 : Changed front end component variables to JS style, fixed some warnings.

### CODE REVIEW

File: frontend/src/components/Histogram.jsx [hunk 1]
```diff
@@ -29,48 +29,47 @@ export default class Histogram extends React.Component {
     console.log("Fetching case data");
 
     // if there is not an available start or end date in the data use this default ones
-    const initial_date =
-      this.props.initial_date != null ? this.props.initial_date : "2020-02-20";
-    const maximum_date =
-      this.props.maximum_date != null ? this.props.maximum_date : "2020-06-23";
+    const initDate =
+      this.props.initialDate != null ? this.props.initialDate : "2020-02-20";
+    const maximDate =
```
Comment (jemrobinson) [inline frontend/src/components/Histogram.jsx]: Suggestion: maybe something like `dateInit` and `dateMaxim` would make it more obvious that these belong together?

File: frontend/src/components/Histogram.jsx [hunk 2]
```diff
@@ -29,48 +29,47 @@ export default class Histogram extends React.Component {
...
-        : this.props.first_restrictions_date;
-    const counterfactual_lockdown_date =
-      this.props.counterfactual_lockdown_date != null
-        ? convert(this.props.counterfactual_lockdown_date)
-        : this.props.lockdown_date;
+    const counterfactualFirstRestrictions =
+      this.props.counterfactualFirstRestrictionsDate != null
+        ? convert(this.props.counterfactualFirstRestrictionsDate)
+        : this.props.firstRestrictionsDate;
+    const counterfactualLockdownDate =
```
Comment (jemrobinson) [inline frontend/src/components/Histogram.jsx]: Similarly here: maybe better as `counterfactualDateLockdown` and `counterfactualDateFirstRestrictions`?

File: frontend/src/components/Histogram.jsx (hunk 1 above)
Comment (crangelsmith) [reply frontend/src/components/Histogram.jsx]: ok will do

File: frontend/src/components/Histogram.jsx (hunk 2 above)
Comment (crangelsmith) [reply frontend/src/components/Histogram.jsx]: Done

File: frontend/src/components/Histogram.jsx (hunk 1 above)
Comment (crangelsmith) [reply frontend/src/components/Histogram.jsx]: Done

**Pull request merged**